types/node-hook/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for node-hook 1.0
+// Project: https://github.com/bahmutov/node-hook#readme
+// Definitions by: Nathan Hardy <https://github.com/nhardy>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/node-hook/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me